src/test/java/io/vertx/ext/asyncsql/PostgreSQLSslConfigurationTest.java

```diff
@@ -86,12 +103,13 @@ public void testWrongSslConfiguration(TestContext context) {
     });
   }
 
+  /*
   @Test
   public void testNoSslConfiguration(TestContext context) {
     Async async = context.async();
     client = createClient(vertx,
       new JsonObject()
-        .put("host", System.getProperty("db.host", "localhost"))
+        .put("host", SQLTestBase.DB_HOST)
         .put("port", Integer.parseInt(System.getProperty("dbssl.port", "54321")))
```

Comment:
Isn't `SQLTextBase.POSTGRESQL_SSL_PORT` missing here?

Reply:
I don't think so as the port wasn't used before, that's how the test works